src/compiler/types.ts

```diff
@@ -10189,7 +10224,78 @@ export type PragmaArgumentType<KPrag extends keyof ConcretePragmaSpecs> = Concre
     : never;
 
 /** @internal */
-export type ConcretePragmaSpecs = typeof commentPragmas;
+export interface ConcretePragmaSpecs {
+    readonly "reference": {

```

Comment:
Still wish we could do something simpler here, but we also don't really touch this that much. (I'm personally okay with it becoming a little less safe just to not have this duplication)